tests/unit/package_managers/test_pip.py

```diff
@@ -2749,10 +2749,8 @@ def test_process_existing_package_without_any_distributions(
             )
         else:
             assert str(exc_info.value.solution) == (
-                "It seems that this version does not exist or isn't published as an"
-                " sdist.\n"
-                "Try to specify the dependency directly via a URL instead, for example,"
-                " the tarball for a GitHub release.\n"
+                "It seems that this version does not exist or isn't published as an sdist.\n"
+                "Try to specify the dependency directly via a URL instead, for example, the tarball for a GitHub release.\n"
```

Comment:
This adjustment also needs to be in a different commit in this series.

Reply:
This diff chunk should not be in this commit (it is actually **correctly** applied in "toil(package managers:pip): cosmetics and types", regardless of hash - that's where it's *always* been 🤷), and I can't make it go away.

It does **NOT** appear for me locally, nor does it show up for me locally if I do completely new fresh clone of my fork.

Reply:
In fact, if I open the file list diff for that specific commit (where the diff shows that incorrect chunk), use the "..." menu to "View" 'pip.py' as a file, then click the "Edit in place" button - the content is correct as expected i.e. **NOT** what appears in that diff.

Reply:
I've been looking at the wrong FILE this whole time - the chunk is in 'test_pip.py', not 'pip.py' :skull:

Reply:
Fixed